sql/core/src/test/scala/org/apache/spark/sql/SQLQueryTestSuite.scala

```diff
@@ -252,6 +256,10 @@ class SQLQueryTestSuite extends QueryTest with SharedSQLContext {
     loadTestData(localSparkSession)
     testCase match {
       case udfTestCase: UDFTestCase => registerTestUDF(udfTestCase.udf, localSparkSession)
+      case _: PgSQLTestCase =>
+        // booleq/boolne used by boolean.sql
```

Comment:
`booleq`/`boolne` adapted it from PostgreSQL:
https://github.com/postgres/postgres/blob/97c39498e5ca9208d3de5a443a2282923619bf91/src/backend/utils/adt/bool.c#L229-L245